### PR TITLE
Bitonic point

### LINKED ISSUE #1459: Implement bitonic point algorithm in ruby
I would like to work on this issue.

### PR DESCRIPTION
**Pull request by AnshitaVishwa**
Fixes #1459 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Implemented bitonic point algorithm in ruby language.

#### Languages Used:
- Ruby

#### Files Added:
- Bitonic_Point.rb

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Bitonic_Point/Bitonic_Point.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        if arr[mid_point] < arr[mid_point + 1]
+            return bitonic(arr, mid_point + 1, right_end)
+        else
+            return bitonic(arr, left_end, mid_point - 1)
+        end
+    end
+    return -1
+end
+
+arr = [1, 2, 3, 4, 5, 9, 6, 5, 2, 1]
```
Comment (MastersAbh) [inline Bitonic_Point/Bitonic_Point.rb]: Please take dynamic inputs

File: Bitonic_Point/Bitonic_Point.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+=begin
+BITONIC POINT
+Given an array. The task is to find the bitonic point of the array.
+The bitonic point in an array is the index before which all the numbers
+are in increasing order and after which, all are in decreasing order.
+=end
+def bitonic(arr , left_end, right_end)
```
Comment (MastersAbh) [inline Bitonic_Point/Bitonic_Point.rb]: blank line before function

**Comment by AnshitaVishwa** [comment]
@MastersAbh I have made the required changes. Please Have a look!
 thanks!!

File: Bitonic_Point/Bitonic_Point.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
+=begin
+BITONIC POINT
+Given an array. The task is to find the bitonic point of the array.
+The bitonic point in an array is the index before which all the numbers
+are in increasing order and after which, all are in decreasing order.
+=end
+
+def bitonic(arr , left_end, right_end)
```
Comment (marwahmanbir) [inline Bitonic_Point/Bitonic_Point.rb]: Remove space before comma. Rest looks good.

**Comment by AnshitaVishwa** [comment]
@marwahmanbir I have made the changes. Please have a look!
Thanks!!

**Comment by MastersAbh** [comment]
@marwahmanbir please have a look.

**Pull request merged**